definitions/ext-solarwinds_server/definition.yml

```diff
@@ -0,0 +1,27 @@
+domain: EXT
+type: SolarWinds_Server
```

Comment:
Hi, could the type be uppercased to keep it consistent with other entity type definitions?

Reply:
Change done